heft-plugins/heft-webpack5-plugin/src/WebpackConfigurationLoader.ts

```diff
@@ -2,72 +2,105 @@
 // See LICENSE in the project root for license information.
 
 import * as path from 'path';
+import type * as TWebpack from 'webpack';
 import { FileSystem } from '@rushstack/node-core-library';
-import type * as webpack from 'webpack';
-import type { IBuildStageProperties, ScopedLogger } from '@rushstack/heft';
+import type { IScopedLogger, IHeftTaskSession, HeftConfiguration } from '@rushstack/heft';
 
-import { IWebpackConfiguration } from './shared';
+import type { IWebpack5PluginOptions } from './Webpack5Plugin';
+import type { IWebpackConfiguration } from './shared';
+
+interface ILoadWebpackConfigurationOptions extends IWebpack5PluginOptions {
+  taskSession: IHeftTaskSession;
+  heftConfiguration: HeftConfiguration;
+  webpack: typeof TWebpack;
+}
 
 /**
  * See https://webpack.js.org/api/cli/#environment-options
  */
 interface IWebpackConfigFunctionEnv {
```

Comment:
This should get exported. That way someone can stick a JSDoc comment in their `webpack.config.js` file and get intellisense for the function params.